core/src/scss/utilities/functions/z-index/_z-index.scss

```diff
@@ -0,0 +1,18 @@
+@charset "UTF-8";
+
+// z($level)
+//
+// A function to help us get the z-index from the level from the map
+//
+// $level - the name of the z-index level
+//
+// Style guide: Functions.z-index.level
+//
+
+@function z($level) {
+  $level: "default";
```

Comment:
This no worky. This will set the passed in parameter to "default" on every call. What you are looking for is:
`@function z($level: 'default') {`

Reply:
Ooh! Thanks!